### ISSUE TITLE
Scope/Assest Selection in Vulnarability Details

### ISSUE THREAD

**Issue by maffmeier** [pwndoc/pwndoc#51]
Why was the asset selection removed? I think this feature was not a bad thing. Can someone explain the situation in more detail?

https://github.com/pwndoc/pwndoc/commit/83e35e370bd7228295e5069bb9e49d4a85491025#diff-6763a535c4cc6a1284134e12c9532d31ac349ac59b0609ac549fe11f12cc3016

**Comment by ConstantinTi** [comment]
I do care about this as well! Any answer is much appreciated!

**Comment by shogunbay** [comment]
I can also agree here. The feature was really useful.

**Comment by yeln4ts** [comment]
It wasn't removed, it just changed from a multiple chips input to an HTML editor. It's easier to copy/paste (especially if you have a large number of assets) and allows much more liberty.

**Comment by maffmeier** [comment]
Wouldn't it be a cool feature to associate this part with the imported scopes?

**Comment by yeln4ts** [comment]
Yes but it wasn't the case before, I wanted to do it that's why the title was misleading but it wasn't a «feature» yet.
What I have in mind would be to find a way to add some kind of suggestions in the editor to add scope parts or maybe a select input allowing to add them to the editor.

**Comment by ConstantinTi** [comment]
> 
> 
> It wasn't removed, it just changed from a multiple chips input to an HTML editor. It's easier to copy/paste (especially if you have a large number of assets) and allows much more liberty.

I actually do agree with that. I think, that your suggestion with the combination of the editor and some sort of select input.
So you could still be as free as you are now, but keep things very simple with the select input.
